packages/leafygreen-provider/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import LeafyGreenProvider from './LeafyGreenProvider';
 import { useUsingKeyboardContext } from './UsingKeyboardProvider';
+import { useBaseFontSize } from './BaseFontSizeProvider';
```

Comment:
```suggestion
export { useBaseFontSize } from './BaseFontSizeProvider';
```

Ditto the other exports.

Reply:
IIRC doing this breaks something with our current build. I don't particularly remember why, but I know there's a reason we haven't been doing this.

Reply:
I think latest version of TS fixed it, we're doing that in MongoNav now.

Reply:
:+1 fixed